spec/requests/api/v1/attendances_spec.rb

```diff
@@ -44,10 +64,8 @@
     end
 
     context 'when viewed by an admin' do
-      before do
-        admin = create(:admin)
-        sign_in admin
```

Comment:
Does this not break the test?  I would assume we have to use the devise test helpers to sign the admin in in order for them to see the right information - if not, we might have a security hole O_O

Reply:
Not at all, it's in the way RSpec works. Since the wrapping describe block has the line

`before { sign_in logged_in_user }`

then the sign_in action will be executed for each example in each context, so all we need to do is override who the `logged_in_user` is when needed, hence the line

`let!(:logged_in_user) { create(:admin) }`

For the admin context

Reply:
Oh DUH.  Thank you for reminding me to read with my eyeballs and for comprehension :/